NewRelic.LogEnrichers.sln

```diff
@@ -40,7 +40,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewRelic.LogEnrichers.NLog"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewRelic.LogEnrichers.NLog.Tests", "src\NLog\NewRelic.LogEnrichers.NLog.Tests\NewRelic.LogEnrichers.NLog.Tests.csproj", "{1FF12FD5-7282-4CB0-A8D8-D213A05658B1}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NewRelic.LogEnrichers.NLog.Examples", "examples\NewRelic.LogEnrichers.NLog.Examples\NewRelic.LogEnrichers.NLog.Examples.csproj", "{078EB9EE-92DF-4989-9E6D-679A25C918AF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewRelic.LogEnrichers.NLog.Examples", "examples\NewRelic.LogEnrichers.NLog.Examples\NewRelic.LogEnrichers.NLog.Examples.csproj", "{078EB9EE-92DF-4989-9E6D-679A25C918AF}"
```

Comment:
Just curious why this changed?

Reply:
Not sure, actually.  hmmmmmm.

Reply:
`9A19103F-16F7-4668-BE54-9A1E7A4F7556` is the correct project type guid for this project because it is an SDK-style project, so this change is innocuous. When the project was originally created it looks like it got the generic C# project guid. I believe Visual Studio has a tendency to automatically update it if you open the project file or something...

https://github.com/dotnet/project-system/blob/master/docs/opening-with-new-project-system.md